client/executor/common/src/sandbox.rs

```diff
@@ -23,21 +23,18 @@
 use crate::{
 	error::{Error, Result},
 	util,
+	wasmer_backend::{instantiate_wasmer, invoke_wasmer, wasmer_new_memory, WasmerBackend},
+	wasmi_backend::{instantiate_wasmi, invoke_wasmi},
 };
-use codec::{Decode, Encode};
+use codec::Decode;
 use sp_core::sandbox as sandbox_primitives;
 use sp_wasm_interface::{FunctionContext, Pointer, WordSize};
 use std::{collections::HashMap, rc::Rc};
-use wasmi::{
-	memory_units::Pages, Externals, ImportResolver, MemoryInstance, Module, ModuleInstance,
-	RuntimeArgs, RuntimeValue, Trap, TrapKind,
-};
+use wasmi::{memory_units::Pages, MemoryInstance, RuntimeValue, Trap, TrapKind};
```

Comment:
We don't want to import stuff from wasmi in this modue. It should be independent of any execution engine. You need to add abstractions so that becomes possible.

Reply:
Agree. Doing that right now